src/main/resources/org/mule/extensions/smart/connector/module-health check.xml

```diff
@@ -21,21 +21,21 @@
            http://www.mulesoft.org/schema/mule/avio-logger http://www.mulesoft.org/schema/mule/avio-logger/current/mule-avio-logger.xsd">
 	
 	
-	<property name="attributes"  type="string" order="1" defaultValue="#[attributes]" summary="Should always be the attributes predefined var"/>
-	<property name="environment" type="string" order="2" defaultValue="#[p('env')]"   example="dev" summary="Environment Name"/>
-	<property name="logCategory" type="string" order="3" displayName="log category"   example="com.avioconsulting.mule.healthcheck" summary="Log category" use="REQUIRED"/>
-	<property name="extSysFlowNames" type="string" order="4" displayName="external systems"   example="flow-name-1,flow-name-2" summary="external systems flow names" use="OPTIONAL"/>
+	<property name="attributes"  type="string" order="1" displayName="Attributes" example="${attributes}" summary="Should always be the attributes predefined var" defaultValue="#[attributes]" use="OPTIONAL"/>
+	<property name="environment" type="string" order="2" displayName="Environment" example="dev" summary="Environment Name" defaultValue="#[p('env')]" use="OPTIONAL"/>
+	<property name="logCategory" type="string" order="3" displayName="Log category"   example="com.avioconsulting.mule.healthcheck" summary="Log category" use="REQUIRED"/>
+	<property name="extSysFlowNames" type="string" order="4" displayName="External systems" example="flow-name-1,flow-name-2" summary="external systems flow names" use="OPTIONAL"/>
+	<property name="healthCheckAppName" type="string" order="5" displayName="Logger app name" example="${app.name}" summary="Environment Name" defaultValue="health-check-api" use="OPTIONAL"/>
+	<property name="healthCheckAppVersion" type="string" order="6" displayName="Logger app version" example="${app.version}" summary="Environment Name" defaultValue="3.0.2" use="OPTIONAL"/>
```

Comment:
I think this is supposed to be the name of the app using the health check API. If so why are default values set to health check API name and version? 

Display names "Logger app name/version" and XML element name healthCheckAppName/Version doesn't look consistent. What is the purpose of these attributes?

Reply:
Agree with @manikmagar , the default values here should be something like ${app.name} ${api.version} or whatever we are using as the default properties in apps that will consume this.  This should be similar what is the default value for environment above.  

Also the summary for these should not be "Environment Name"